templates/yandex-music.sh

```diff
@@ -0,0 +1,12 @@
+#!/bin/bash
+
+XDG_CONFIG_HOME=${XDG_CONFIG_HOME:-~/.config}
```

Comment:
Put variables in `"` please. Here and two places below

Reply:
Fix in new commit